manifests/k8s-deployment-template.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: external-mdns
+spec:
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: external-mdns
+  template:
+    metadata:
+      labels:
+        app: external-mdns
+    spec:
+      securityContext:
+        runAsUser: 65534
+        runAsGroup: 65534
+        runAsNonRoot: true
+      hostNetwork: true
+      serviceAccountName: external-mdns # Assicurati che questo service account esista
```

Comment:
The documentation and code comments in this repo are all written in English. Would it be too much to ask for you to change all of these new comments to English for the sake of consistency?

Reply:
LGTM